Ron-News-Front/src/components/Votes.jsx

```diff
@@ -1,52 +1,76 @@
 import { useEffect, useState } from "react";
+import { patchArticleVotes } from "../api-functions/patchArticleVotesByArticleId";
 
-export const Votes = ({votesData}) => {
+export const Votes = ({votesData, article_id}) => {
     const [votes, setVotes] = useState(0);
```

Comment:
You could assign votes it's initial amount here by replacing 0 with the vote amount